### ISSUE TITLE
Option to hide share button iOS

### ISSUE THREAD

**Issue by ItzSteveHuh** [brave/brave-browser#38858]
### Platforms

iOS

### Description

The share button is too close to the back button, one could easily hit the share button by mistake and if your not care you may accidentally share a page to someone. It would be great if there was an option to hide it.

**Comment by ItzSteveHuh** [comment]
Bump is there someone who can edit this to add the appropriate labels? I know this wouldn't be a priority but hopefully something that can be added down the road.

**Comment by ItzSteveHuh** [comment]
Stale

**Comment by andrew171717** [comment]
As a long term Chrome user who is looking for an alternative browser to switch to, this is the only functional holding me back from switching. As silly as that might sound, the share button’s location is extremely frustrating from my perspective. I’ve accidentally tapped it multiple times when trying to browse forward. Please reconsider adding an option in the settings to hide the share button. When hidden, you could show the forward arrow greyed out in its place (this is how mobile iOS Chrome functions).

**Comment by bsclifton** [comment]
cc: @jamesmudgett

**Comment by LuciferWPF910** [comment]
I agree that the positioning of the share button on (seemingly all) browsers on iPhones is poor. 

It's bad UX, you don't place a button like that next to something that you actively know users will fat thumb.

I also conceptually find it a bit illogical. I feel (granted this is subjective) as if the back and share should be on opposite sides at least considering there is no forward button [meaning a redo of the undo]

Now what can be done about this from your side I'm totally unsure about-

Considering every browser I've used on iPhone has had an identical layout it **suggests** to me that this may be something baked in for iPhones.

However if anything at all can be done even if it's just a slider that makes the button faded out and un-clickable like others have already suggested would be a worthwhile development.

**Comment by ItzSteveHuh** [comment]
**Bump** — can we get an update on this?

Please consider adding an option to hide the Share button on iOS.

A similar option was recently added to desktop:
https://github.com/brave/brave-browser/issues/49180

On iOS the Share button also occupies the same location as the **Forward navigation button**. If there is forward history, the Forward button appears; otherwise the Share button is shown in that same position.

Because these two different actions share the same location, it's easy to tap Share by mistake when expecting the Forward button. An option to hide the Share button would help prevent accidental taps and improve UI consistency.

**Comment by pranavmangal** [comment]
As others have mentioned, Brave on iOS currently uses a single button for forward and share and switches the button's function depending on the current position in history. This is generally subpar UX because it conflicts with muscle memory.

The [Orion](https://orionbrowser.com/platforms/ios) browser handles this better by having permanent forward and share buttons:

![Image](https://github.com/user-attachments/assets/ee861f51-76fd-4477-ab3a-0f73d72f6a96)

If Brave doesn't want to have a customizable tab bar, atleast it can consider adopting a similar layout to avoid confusion.
